kube-monitoring/charts/values.yaml

```diff
@@ -7,10 +7,55 @@ global:
 ## kube-prometheus-stack configuration scoped to kube-monitoring
 kubeMonitoring:
 
-  ## Create default rules for monitoring the cluster
+  ## Create open source community rules for monitoring the cluster
   ##
   defaultRules:
-    create: false
+    create: true
+    rules:
+      alertmanager: false
+      etcd: false
+      configReloaders: false
+      general: false
+      k8sContainerCpuUsageSecondsTotal: false
+      k8sContainerMemoryCache: false
+      k8sContainerMemoryRss: false
+      k8sContainerMemorySwap: false
+      k8sContainerResource: false
+      k8sContainerMemoryWorkingSetBytes: false
+      k8sPodOwner: true
+      kubeApiserverAvailability: false
+      kubeApiserverBurnrate: false
+      kubeApiserverHistogram: false
+      kubeApiserverSlos: false
+      kubeControllerManager: false
+      kubelet: true
+      kubeProxy: false
+      kubePrometheusGeneral: true
+      kubePrometheusNodeRecording: true
+      kubernetesApps: false
+      kubernetesResources: false
+      kubernetesStorage: false
+      kubernetesSystem: false
+      kubeSchedulerAlerting: false
+      kubeSchedulerRecording: false
+      kubeStateMetrics: false
+      network: false
+      node: false
+      nodeExporterAlerting: false
+      nodeExporterRecording: true
+      windows: false
+
+      ## Enable useful alerting rules for self-monitoring Prometheus
+      prometheus: true
+      prometheusOperator: true
+
+    ## Additional labels for PrometheusRule alerts. E.g. support_group and service.
+    additionalRuleLabels: {}
+
+  ## Create dashboards for self monitoring Prometheus
+  ##
+  dashboards:
+    create: true
```

Comment:
what kind of dashboards are we talking here?

Reply:
The dashboard template is part of this PR.